text/0000-landing-page.md

```diff
@@ -0,0 +1,53 @@
+# Meta
+[meta]: #meta
+- Name: Create a community landing page
+- Start Date: 1-6-2020
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+People interested in the project don't have a central place to go for information. This RFC proposes the creation of a community or "landing page" repo at `buildpacks/buildpacks` with high level information about the project including key personel, governance, guidelines for contributions, meeting schedules, etc.
+
+**Outcomes**
+[outcomes]: #outcomes
+
+ - New contributors feel welcomed.
+ - A new contributor or user of the project can easily find what they want, join the slack, WG meetings, and immediately contribute pull-requests, RFCs, and issues.
+
+# Motivation
+[motivation]: #motivation
+
+A major part of the success of the CNB project will necessitate growing contributors beyond the current core. As new people learn about the project and want to use it or contribute, we need a friendly central landing place where all the information _about_ the project is housed.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This RFC proposes the creation of a project landing page that will be house at `buildpacks/buildpacks`.
```

Comment:
I guess one problem with it being a "landing page" is that it won't naturally float to the top (because it probably won't have frequent commits). This is probably a "drawback", but I still think it's worth having.

Reply:
I think we could “pin” it in the project so it would show up at the top, but thats not a perfect solution

Reply:
See above re `buildpacks/community`